cdap-app-fabric/src/main/java/io/cdap/cdap/internal/capability/CapabilityApplier.java

```diff
@@ -82,26 +95,32 @@
   private static final ProgramTerminator NOOP_PROGRAM_TERMINATOR = programId -> {
     // no-op
   };
+  private static final int numThreads = 5;
```

Comment:
constant should be all caps. Also, use a more meaningful naming than numThreads (what is this number of threads for?)

Reply:
Also, why is it not configurable?

Reply:
Renamed as autoInstallThreads and made configurable.